test/bigtable/row.js

```diff
@@ -591,8 +564,7 @@ describe('Bigtable/Row', function() {
       };
 
       row.get(function(err, row_, apiResponse) {
-        assert(err instanceof Error);
-        assert.strictEqual(err.message, 'Unknown row: ' + ROW_ID);
+        assert(err instanceof Row.RowError);
```

Comment:
I think this might be what I said to do (check that its an instance of `RowError`), but it's also necessary to check that it was created the right way. So just another stub that checks the arguments sent to `Row.RowError` and whatever it returns is what the `err` argument to the callback is. So you don't have to check that it's a `RowError`-- it can just be whatever you return from the stub.